src/file/file.service.spec.ts

```diff
@@ -7,6 +7,8 @@ import { ServerError, ClientError } from '../utils/errors/application.error';
 import { FileExistsWithSameName, KeyAlreadyExistsError, FileNotFoundError } from '../utils/errors/client.error';
 import { FolderNotFoundError } from '../utils/errors/folder';
 import { userInfo } from 'os';
```

Comment:
`import { userInfo } from 'os';` should be before local imports like `import { FolderNotFoundError } from '../utils/errors/folder';`